templates/manifest.json

```diff
@@ -1,106 +1,134 @@
+            },
+            "skipOptionalDeps": true
+        },
+        {
+            "name": "project_puppeteer_crawler_js",
+            "label": "PuppeteerCrawler in JavaScript",
+            "category": "javascript",
+            "technologies": ["crawlee", "puppeteer"],
+            "description": "Standard and up to date template for developing with Crawlee's PuppeteerCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_puppeteer_crawler_js.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 4096,
+                "timeoutSecs": 3600
+            }
+        },
+        {
+            "name": "project_playwright_crawler_js",
+            "label": "PlaywrightCrawler in JavaScript",
+            "category": "javascript",
+            "technologies": ["crawlee", "playwright"],
+            "description": "Standard and up to date template for developing with Crawlee's PlaywrightCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_playwright_crawler_js.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 4096,
+                "timeoutSecs": 3600
+            }
+        },
+        {
+            "name": "project_cheerio_crawler_ts",
+            "label": "CheerioCrawler in TypeScript",
+            "category": "typescript",
+            "technologies": ["crawlee", "cheerio"],
+            "description": "Standard and up to date template for developing with Crawlee's CheerioCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_cheerio_crawler_ts.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 512,
+                "timeoutSecs": 3600
+            },
+            "skipOptionalDeps": true
+        },
+        {
+            "name": "project_puppeteer_crawler_ts",
+            "label": "PuppeteerCrawler in TypeScript",
+            "category": "typescript",
+            "technologies": ["crawlee", "puppeteer"],
+            "description": "Standard and up to date template for developing with Crawlee's PuppeteerCrawler.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/project_puppeteer_crawler_ts.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 4096,
+                "timeoutSecs": 3600
+            }
+        },
+        {
+            "name": "project_playwright_crawler_ts",
+            "label": "PlaywrightCrawler in TypeScript",
+            "category": "typescript",
+            "technologies": ["crawlee", "playwright"],
+            "description": "Standard and up to date template for developing with Crawlee's PlaywrightCrawler.",
```

Comment:
Standard and up-to-date template for developing with Crawlee's PlaywrightCrawler.